packages/prisma/seed-insights.ts

```diff
@@ -75,44 +143,146 @@ async function main() {
         },
       ],
     });
-    await prisma.team.update({
-      where: {
-        id: insightsTeam.id,
-      },
-      data: {
-        members: {
-          connect: {
-            userId_teamId: {
-              userId: insightsAdmin.id,
-              teamId: insightsTeam.id,
+    if (!insightsTeam) {
+      insightsTeam = await prisma.team.create({
+        data: {
+          name: "Insights",
+          slug: "insights-team",
+        },
+      });
+    }
+
+    if (insightsAdmin && insightsTeam) {
+      await prisma.team.update({
+        where: {
+          id: insightsTeam.id,
+        },
+        data: {
+          members: {
+            connect: {
+              userId_teamId: {
+                userId: insightsAdmin.id,
+                teamId: insightsTeam.id,
+              },
             },
           },
         },
-      },
-    });
+      });
+    }
   }
-  await prisma.team.update({
+
+  let teamEvents = await prisma.eventType.findMany({
     where: {
-      id: insightsTeam.id,
+      teamId: insightsTeam.id,
     },
-    data: {
-      members: {
-        connect: {
-          userId_teamId: {
-            userId: insightsUser.id,
-            teamId: insightsTeam.id,
-          },
-        },
-      },
+  });
+
+  const userSingleEventsAdmin = await prisma.eventType.findMany({
+    where: {
+      userId: insightsUser.id,
     },
   });
 
-  let teamEvents = await prisma.eventType.findMany({
+  const userSingleEvents = await prisma.eventType.findMany({
     where: {
-      teamId: insightsTeam.id,
+      userId: insightsUser.id,
     },
   });
 
+  if (userSingleEventsAdmin.length === 0 && userSingleEvents.length === 0) {
```

Comment:
I added bookins and event types that belong to team members but they don't belong to the team.